pkg/ddl/reorg_util.go

```diff
@@ -91,9 +91,18 @@ func initJobReorgMetaFromVariables(ctx context.Context, job *model.Job, tbl tabl
 		}
 	})
 
+	var autoConc, autoMaxNode int
+	if kerneltype.IsNextGen() {
+		factors, err := dxfhandle.GetScheduleTuneFactors(ctx, sctx.GetStore().GetKeyspace())
+		if err != nil {
+			return err
+		}
```

Comment:
Maybe add this factors to below log?

Reply:
https://github.com/pingcap/tidb/pull/63517/commits/cf01f958d64a919b6d94bec06ff4211dea3df06a